CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+# 105.0.0
+
+In this major release we removed the deprecated themers: MDCTextFieldTypographyThemer, MDCOutlinedButtonThemer, and added a new MDCBaseTextArea class under TextControls.
+
+## Breaking changes
+
+Deleted MDCTextFieldTypographyThemer and MDCOutlinedButtonThemer classes. Please use the theming extensions.
+Learn more at https://github.com/material-components/material-components-ios/blob/develop/docs/theming.md#migration-guide-themers-to-theming-extensions
+
+## New features
+
+A new BaseTextArea component that leverages UITextView to provide multi-line text input have been created at components/TextControls/src/BaseTextAreas/.
+
+## Component changes
+
+### ActionSheet
+
+* [Migrates Action sheet to MDCAvailability. (#9733)](https://github.com/material-components/material-components-ios/commit/652077c779b06fc6b44e9bbae237f47e083c053c) (Bryan Oltman)
+
+### ActivityIndicator
+
+* [Migrates activity indicator to MDCAvailability. (#9735)](https://github.com/material-components/material-components-ios/commit/28a7fac63f7ee5f1787ab56190c8a579532810d6) (Bryan Oltman)
+
+### BottomNavigation
+
+* [Migrates Bottom Navigation to MDCAvailability. (#9717)](https://github.com/material-components/material-components-ios/commit/02efdd188990305ff298fbad25fa48634974b1b4) (Randall Li)
+
+### Buttons
+
+* [Fix tvOS target bug. (#9705)](https://github.com/material-components/material-components-ios/commit/95518f48d4fe6c10e13cbdcfcc3a1c863697298d) (Randall Li)
+* [MDCButton. (#9667)](https://github.com/material-components/material-components-ios/commit/1d8d647f1f534de4366c8722992b67858d7ba444) (copybara-service[bot])
+* [Delete deprecated MDCOutlinedButtonThemer (#9688)](https://github.com/material-components/material-components-ios/commit/1118f14124858fe9a258b0971393e33b3966d2bf) (Bryan Oltman)
+
+### Cards
+
+* [Migrates cards to MDCAvailability. (#9728)](https://github.com/material-components/material-components-ios/commit/b677a3629e180a89f1e6a15256b45790ca794af5) (Randall Li)
+
+### Chips
+
+* [Migrates Chips to MDCAvailability. (#9713)](https://github.com/material-components/material-components-ios/commit/33fbdfc3be62b4ff78951898b9abe6fc59041db4) (Randall Li)
+
+### Dialogs
+
+* [Add snapshot tests for additional dialog configurations. (#9646)](https://github.com/material-components/material-components-ios/commit/e2bff768df18fc24f91857447ddb8992071e778f) (copybara-service[bot])
```

Comment:
Why some PRs are not attributed to the author(s)?

Reply:
they were copied via the copybara bot from internal CLs which is why they show like that.

Reply:
Changed to your name in these PRs